src/webgpu/shader/execution/builtin/countLeadingZeros.spec.ts

```diff
@@ -0,0 +1,271 @@
+export const description = `
+Execution Tests for the 'countLeadingZeros' builtin function
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+import { i32, i32Bits, TypeI32, u32, TypeU32, u32Bits } from '../../../util/conversion.js';
+
+import { Config, run } from './builtin.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('integer_builtin_functions,countLeadingZeros_unsigned')
+  .uniqueId('xxxxxxxxxxxxxxxx')
+  .specURL('https://www.w3.org/TR/2021/WD-WGSL-20210929/#integer-builtin-functions')
```

Comment:
Note: this matches the URL of all the other test plans, which will also need bulk updating.

Reply:
I don't feel particularly strongly about this, since as you pointed out it matches the existing tests and likely needs a bulk update across the code base.

Reply:
Let's land these then, and I'll do a bulk update of all of these in a single atomic commit.